gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-container/src/main/resources/META-INF/spring.factories

```diff
@@ -1,2 +1,4 @@
 io.gravitee.node.api.healthcheck.Probe=\
-    io.gravitee.gateway.standalone.healthcheck.HttpServerProbe
\ No newline at end of file
+    io.gravitee.gateway.standalone.healthcheck.HttpServerProbe
+io.gravitee.node.container.ContainerInitializer=\
+    io.gravitee.gateway.standalone.APIMGatewayProductInitializer
```

Comment:
Is it necessary?

Reply:
That is how it is Initializing bean work in Node I just kept the way it is done.

Reply:
It is mandatory... simple bean don't work.